systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthAuthorizationIsolatedST.java

```diff
@@ -687,6 +687,7 @@ void setUp(ExtensionContext extensionContext)  {
     void tearDown(ExtensionContext extensionContext) throws Exception {
         // delete keycloak before namespace
         KeycloakUtils.deleteKeycloak(Constants.INFRA_NAMESPACE, clusterOperator.getDeploymentNamespace());
+        KeycloakUtils.deletePostgres(Constants.INFRA_NAMESPACE);
```

Comment:
Same as for the deploy -> `deleteKeycloakAndPostgres`

Reply:
While I strongly disagree here, I will do it as you want.